fiat-core/src/main/java/com/netflix/spinnaker/fiat/model/resources/Permissions.java

```diff
@@ -106,8 +106,7 @@ public boolean isEmpty() {
                .collect(Collectors.toSet());
   }
 
-  @VisibleForTesting
-  protected List<String> get(Authorization a) {
+  public List<String> get(Authorization a) {
```

Comment:
This makes writing the migration easier as there isn't a easy way now to get a list of all groups who have READ or WRITE access.